extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -84,6 +90,9 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
     if (!tagData) {
       return;
     }
+    tagData['rightEdgeOfScreentoTagDistance'] =
+      document.getElementsByTagName('amp-story-shopping-attachment')[0]
+        .clientWidth - this.element.offsetLeft;
```

Comment:
Does this calculation need to also take into account the width of the tag in order to determine the distance?

Reply:
What Corey said, + RTL support :))
Feel free to look into what can be re-used from existing tooltips. Also for anything accessibility. Have we considered building on top of the existing tooltips to get all of this for free?

Reply:
Will add RTL support once PM discussion and approval on currency format is complete. See my comment on https://github.com/ampproject/amphtml/pull/37187#discussion_r768276620

Reply:
This looks unrelated to the currency string. The shopping tag should always be positioned correctly for both RTL and LTR.

Reply:
I will take a look into RTL for this ticket after the specs ticket is merged #37186

Reply:
RTL Support was added in with this PR: #37249

Reply:
also added in the width of the tag calculation